site/src/pages/docs/advanced-spring-boot-integration.mdx

```diff
@@ -0,0 +1,382 @@
+# Using Spring Boot with Armeria
+
+Spring framework provides powerful features which are necessary for building a web application, such as
+dependency injection, data binding, AOP, transaction. Armeria is a lightweight microservices framework,
+its capabilities is comparable to existing full stack web frameworks. It has Rich support for
+[Apache Thrift](https://thrift.apache.org/) and [gRPC](https://grpc.io/), including [the fancy web console](https://armeria.dev/docs/server-docservice)
+that enables you to send Thrift and gRPC requests from a web browser. Armeria supports both HTTP/1 and HTTP/2,
+and these protocols both support [cleartext](https://simple.wikipedia.org/wiki/Cleartext) and TLS cryptographic protocols.
+Armeria uses [event loop](https://en.wikipedia.org/wiki/Event_loop) based on [Netty](https://netty.io/index.html)
+which allows to perform non-blocking I/O operations. Therefore, it shows better performance in certain situations such as media streaming,
+or large file transfer. If you integrate your Spring application with Armeria, you can serve both legacy Spring service
+and any service made of Armeria such as gRPC or Thrift in the same port. Also, you can gradually migrate your services
+starting with those expected to improve performance.
+
+Armeria provides a way to integrate legacy Spring Boot applications with just simple code.
+Additionally, Armeria supports several internal services for monitoring and management purposes.
+You can achieve these convenient properties through bean configuration.
+
+## Integrate Legacy Spring Boot with Armeria
+
+We will plug Armeria into the Spring Boot system. Add the following dependency to your application.
+
+<RequiredDependencies
+  dependencies={[
+    {
+      groupId: 'com.linecorp.armeria',
+      artifactId: 'armeria',
```

Comment:
We don't need this dependency because other dependencies have this as API configuration